### DISCUSSION TITLE
How to Change Sidebar label to red when the link is active

### DISCUSSION THREAD

**Discussion by noblemfd** [ColorlibHQ/AdminLTE#4182] [Q&A] [answered]
In my AdminLTE-V3, I have this sidebar:

      <li class="nav-item menu-open">
            <a href="#" class="nav-link active">
              <i class="nav-icon fas fa-tachometer-alt"></i>
              <p>
                Dashboard
                <i class="right fas fa-angle-left"></i>
              </p>
            </a>
            <ul class="nav nav-treeview">
              <li class="nav-item">
                <a href="./index.html" class="nav-link active">
                  <i class="far fa-circle nav-icon"></i>
                  <p>Dashboard v1</p>
                </a>
              </li>
              <li class="nav-item">
                <a href="./index2.html" class="nav-link">
                  <i class="far fa-circle nav-icon"></i>
                  <p>Dashboard v2</p>
                </a>
              </li>
              <li class="nav-item">
                <a href="./index3.html" class="nav-link">
                  <i class="far fa-circle nav-icon"></i>
                  <p>Dashboard v3</p>
                </a>
              </li>
            </ul>
          </li>

When any of the sidebar label is click (active), I want to change the color to red as shown in the diagram:

![sidebar_activelink](https://user-images.githubusercontent.com/34077129/149498663-039a92f2-f9ea-41d8-ac75-91f60eaf6ed4.PNG)

How do I achive this?

Thanks

**Comment by REJack** [comment, marked as answer]
You can use `sidebar-dark-danger` instead of `sidebar-dark-primary` but the white subitem active will not change to red either. if you want that changed too you need to create a custom style for it.
